noxfile.py

```diff
@@ -50,13 +37,4 @@ def docs(session):
 def test(session):
     session.install("-r", "test-requirements.txt")
 
-    session.run(
-        "coverage",
-        "run",
-        "--omit='*'",
-        "-m",
-        "pytest",
-        "--cov=httpx",
-        "--cov=tests",
-        "--cov-report=term-missing",
-    )
+    session.run("coverage", "run", "-m", "pytest")
```

Comment:
I've been curious why this is run this way, instead of using [`pytest-cov`](https://github.com/encode/httpx/blob/master/test-requirements.txt#L20) directly, as recommended by coverage docs themselves: https://coverage.readthedocs.io/en/v4.5.x/#using-coverage-py

AFAIK, `pytest --cov=httpx --cov=tests --cov-report=term-missing` will run the same as `coverage run -m pytest...`, and not require `--omit='*'`, which excludes everything from `coverage` and ends up using `pytest-cov`'s run anyways.

Reply:
This was something I was doing as it seemed to work to not track coverage relative to the virtualenv. This appears to work though?

Reply:
I'll change it, the only difference is that `coverage` adds `httpx` to the execution context automatically so you don't need to install it before running the test command.

To run with `pytest` directly it's required to install `httpx` before running like that:

```python
    session.install("-e", ".")

    session.run("pytest")
```
Otherwise, the nox execution will fail when looking for `httpx` at the cases where nox is installed globally

Reply:
In that case, one can also run `("python", "-m", "pytest")`
Src: https://docs.pytest.org/en/latest/usage.html#calling-pytest-through-python-m-pytest

Reply:
That was clever :smile:

Reply:
Added these changes at 4ebc0094482c5e760cf79edaf3338924af1d91fe